crates/op-rbuilder/src/generator.rs

```diff
@@ -260,24 +251,19 @@ where
 
         let (tx, rx) = oneshot::channel();
         self.build_complete = Some(rx);
+        let args = BuildArguments {
```

Comment:
I was imagine that it would be PayloadGenerator the one spawning the job instead of the PayloadBuilder impl.

Reply:
I could move this too into the OpPayloadBuilderVanilla. 
I will add config and cancel to the try_build and then see if we could refactor this somehow.

Reply:
> I could move this too into the OpPayloadBuilderVanilla.

Forget about the highlighted line, I could not highlight the correct part of the code since it was not modified.

Reply:
Seems like i read initial message wrong.
You man to move spawning into new_payload_job of BlockPayloadJobGenerator, right?

Reply:
Ideally i would change interface in this place a bit.
I would like to see the following:
Instead of resolving per-job we would me payload resolving logic into `BlockPayloadJobGenerator`
The `BlockPayloadJobGenerator` would coordinate all `BlockPayloadJob` spawned for this payload building and will ultimately return response to `PayloadBuilderService`